src/test/scala/ai/privado/exporter/FeignEgressExportTest.scala

```diff
@@ -55,11 +80,10 @@ class FeignEgressExportTest extends JavaTaggingTestBase {
       |
       |}
       |
-      |""".stripMargin
+      |""".stripMargin)
 
-  "Java code for Feign Client" should {
     "collect egress url from feign client" in {
-      val propertyExporter = new HttpConnectionMetadataExporter(cpg, ruleCache, new AppCache())
+      val propertyExporter = new HttpConnectionMetadataExporter(cpg, ruleCache, appCache)
```

Comment:
we shouldn't be running passes explicitly. The Egress urls will be exported in the outputMap, we need to cross check there, if this has been exported

Reply:
yes right, i'm removing the explicitly called functions.